test/e2e/externalservices/locality_multizone_hybrid.go

```diff
@@ -51,33 +53,41 @@ func ExternalServicesOnMultizoneHybridWithLocalityAwareLb() {
 		globalCP := global.GetKuma()
 
 		// K8s Cluster 1
+		group := errgroup.Group{}
 		zone1 = NewK8sCluster(NewTestingT(), Kuma1, Silent)
-		Expect(NewClusterSetup().
-			Install(Kuma(config_core.Zone,
-				WithIngress(),
-				WithIngressEnvoyAdminTunnel(),
-				WithEgress(),
-				WithEgressEnvoyAdminTunnel(),
-				WithGlobalAddress(globalCP.GetKDSServerAddress()),
-			)).
-			Install(NamespaceWithSidecarInjection(TestNamespace)).
-			Setup(zone1)).To(Succeed())
+		group.Go(func() error {
```

Comment:
Q: Is it possible to create a new goroutine when calling `Setup(cluster)` and perhaps provide a group there? It seems like we could do this, but I may be missing some context. (but maybe it's not required if only legacy tests are using it)

```
                         group := errgroup.Group{}
			err := NewClusterSetup().
				Install(Kuma(config_core.Zone, WithGlobalAddress(globalCP.GetKDSServerAddress()))).
				Install(DemoClientUniversal(
					"zone4-demo-client",
					defaultMesh,
					WithTransparentProxy(true),
				)).
				Install(IngressUniversal(globalCP.GenerateZoneIngressToken)).
				Install(EgressUniversal(globalCP.GenerateZoneEgressToken)).
				Install(TestServerExternalServiceUniversal("external-service-in-zone1", 8080, false)).
				Setup(zone4, group)
```

Or something to avoid remembering about `group.Go`

Reply:
I guess we could do something like `.SetupInGroup(zone4, group)